src/main/java/uk/gov/pay/directdebit/payments/api/CollectPaymentResponse.java

```diff
@@ -26,6 +28,12 @@
     @JsonProperty
     private Long amount;
 
+    @JsonProperty("mandate_id")
+    private String mandateId;
+    
+    @JsonProperty("provider_id")
+    private String providerId;
```

Comment:
There is a `PaymentProviderPaymentId` class to represent this, please can you use that instead of String?